treestats_paper.tex

```diff
@@ -786,6 +803,8 @@ \section*{Duality of site and branch statistics}
 while the second term, the variance of the branch statistic,
 is the contribution of demographic variation, including drift.
 
+\krt{will this section go further?  It just kind of ends...}
+
```

Comment:
Well, we don't have any more content, but do return to this idea in e.g. Fig 3, so could allude to that? Any other ideas?

Reply:
Let me think about this one.